types/db.go

```diff
@@ -67,3 +68,21 @@ type Attribute struct {
 	Key        string        `gorm:"index" json:"key"`
 	Value      string        `json:"value"`
 }
+
+// Plan is a Terraform plan
+type Plan struct {
+	ID               uint           `sql:"AUTO_INCREMENT" gorm:"primary_key" json:"id"`
+	Version          Version        `json:"version"`
+	VersionID        sql.NullInt64  `gorm:"index" json:"-"`
+	Lineage          string         `json:"lineage"`
```

Comment:
I added a Lineage field @mcanevet, that's ok? Or should I put a foreign key instead?
I did not really understand who is the agent who will take care of the insertions to the db so I do not really know what information we will have

Reply:
I'm not sure you will be able to set a foreign key here and if it would be interesting. I think we can let this as it is and change it later if needed.

Reply:
I'm not sure either. We have to discuss this point with @raphink

Reply:
I think it's fine like this.